#13 zoom in/out

promedolum opened the pull request:
Buttons are placed on one view;
Button shape is fixed;
Note to myself: exclude *.pbxproj from commits

promedolum committed:
Basic Zoom Buttons

promedolum committed:
Fixed outlet name, removed "own" bundle identifier and team intended to run on real device.

promedolum committed:
#13 added zoom in/out buttons on single view. fixed shape

kzyryanov commented:
1. Перенеси IBActions после секции Private и пометь комментом: // MARK: - Actions
2. Вынеси "магические" константы 0.5 и 1.5 в константу класса (private static let zoomPercent: CLLocationDegrees = 0.5) и в формулах испльзуй:
```
span.latitudeDelta = region.span.latitudeDelta * (1.0 ± MapViewController.zoomPercent)
span.longitudeDelta = region.span.longitudeDelta * (1.0 ± MapViewController.zoomPercent)
```

promedolum committed:
#13 fix for #issuecomment-286072380

promedolum commented:
Внёс изменения согласно комментарию @kzyryanov

promedolum committed:
CONSTRAINS ARE BROKEN; merging to not to keep up much with origin/master

Merge remote-tracking branch 'origin/master' into 13-zoom-inout

# Conflicts:
#	MapsAOPA.xcodeproj/project.pbxproj
#	MapsAOPA/Classes/MapViewController.swift
#	MapsAOPA/Resourses/Base.lproj/Main.storyboard

promedolum committed:
Merge remote-tracking branch 'origin/master' into 13-zoom-inout

# Conflicts:
#	MapsAOPA.xcodeproj/project.pbxproj
#	MapsAOPA/Classes/MapViewController.swift
#	MapsAOPA/Resourses/Base.lproj/Main.storyboard

promedolum committed:
Merge remote-tracking branch 'origin/13-zoom-inout' into 13-zoom-inout

# Conflicts:
#	MapsAOPA/Classes/MapViewController.swift

promedolum committed:
Removed buttonsView outlet and subview from code, added constraints to buttonView

promedolum committed:
Merge remote-tracking branch 'origin/master' into 13-zoom-inout

# Conflicts:
#	MapsAOPA/Classes/MapViewController.swift
#	MapsAOPA/Resourses/Base.lproj/Main.storyboard

kzyryanov committed:
Merge branch 'master' into 13-zoom-inout

# Conflicts:
#	MapsAOPA/Resourses/Base.lproj/Main.storyboard
